### PR TITLE
feat: detect browser locale

### PR DESCRIPTION
**Pull request by salzig**
so far we ignore the browser provided list of user preferred locales. Maybe it's worth to respect the user hint?

### CODE REVIEW

File: spec/controllers/concerns/locale_detection_sprec.rb [hunk 1]
```diff
@@ -0,0 +1,51 @@
+require 'spec_helper'
```
Comment (JoschkaSchulz) [inline spec/controllers/concerns/locale_detection_sprec.rb:1]: # frozen_string_literal: true 

is missing ;) will be obsolete with ruby 3.4.0 but for now we need it

Comment (salzig) [reply spec/controllers/concerns/locale_detection_sprec.rb:1]: current lived standard is without (at least in this repository, for the specs) :-P

Comment (JoschkaSchulz) [reply spec/controllers/concerns/locale_detection_sprec.rb]: 😄 👍

Comment (JoschkaSchulz) [reply spec/controllers/concerns/locale_detection_sprec.rb]: I mean it's obsolete soon

**Review by JoschkaSchulz — APPROVED** [review]
ah I see, the method is fixed now :)

**Pull request merged**